core/dplug/core/ringbuf.d

```diff
@@ -128,8 +128,12 @@ struct RingBufferNoGC(T)
         T opIndex(size_t index) nothrow @nogc
         {
             // crash if index out-of-bounds (not recoverable)
-            if (index > _count)
-                assert(0);
+            version(D_NoBoundsChecks) {}
+            else
+            {
+                if (index >= _count)
```

Comment:
I'd think, `>=` makes more sense than `>` (as opposed to the original version).

> Previously the maximum allowed index was .length
> that equals actualMaxIndex+1.

Reply:
Er, absolutely!